helix-sitemap.yaml

```diff
@@ -0,0 +1,54 @@
+version: 1
+
+sitemaps:
+  seo:
+    origin: https://www.adobe.com
+    lastmod: YYYY-MM-DD
+    languages:
+      au:
+        source: /hub/query-index-au.json
+        destination: /hub/sitemap/sitemap-au.xml
```

Comment:
will keep sitemaps in `/sitemap/` for now; once we get 👍 I will move them to `/hub/`